dev/benchmarks/platform_channels_benchmarks/android/app/src/main/kotlin/com/example/platform_channels_benchmarks/MainActivity.kt

```diff
@@ -12,15 +12,21 @@ import io.flutter.plugin.common.StandardMessageCodec
 import java.nio.ByteBuffer
 
 class MainActivity: FlutterActivity() {
-
+    private var byteBufferCache : ByteBuffer? = null
```

Comment:
nit: consider adding a comment about why we do this (e.g. what you have in your PR description).